src/com/pty4j/util/PtyUtil.java

```diff
@@ -24,8 +24,8 @@ public static String[] toStringArray(Map<String, String> environment) {
   private static @Nullable File getPreferredLibPtyFolder() {
     String path = System.getProperty(PREFERRED_NATIVE_FOLDER_KEY);
     File dir = path != null ? new File(path) : null;
-    if (dir != null && dir.isAbsolute() && dir.isDirectory()) {
-      return dir;
+    if (dir != null && dir.isDirectory()) {
```

Comment:
LGTM

Maybe let's check that `System.getProperty(PREFERRED_NATIVE_FOLDER_KEY)` is non-empty?

Reply:
Good idea; done